dot/peerset/peerset.go

```diff
@@ -24,7 +25,7 @@ const (
 	// forgetAfterTime amount of time between the moment we disconnect
 	// from a node and the moment we remove it from the list.
 	forgetAfterTime = time.Second * 3600 // one hour
-	// default channel size for peerSet.
+
 	msgChanSize = 100
```

Comment:
A bit out of scope, but do we actually need a buffer channel??

Reply:
We have both `actionQueue` and `resultMsgCh` as buffered channels. In fact, I believe, we don't need this `msgChanSize`, should I open an issue to turn them into unbuffered channels?

Reply:
Maybe open an issue to investigate all buffered channels requirement? You can add that one as a first bullet point?